src/xclim/data/fr.json

```
@@ -1033,6 +1033,18 @@
     "title": "Jours avec neige",
     "abstract": "Nombre de jours où la neige est entre une borne inférieure et supérieure."
   },
+  "HOLIDAY_SNOW_DAYS": {
+    "long_name": "Nombre de jours de neige durant les jours fériés",
+    "description": "Nombre de jours de neige durant les jours fériés.",
+    "title": "Jours de neige durant les jours fériés",
+    "abstract": "Nombre de jours de neige durant les jours fériés."
+  },
+  "HOLIDAY_SNOW_AND_SNOWFALL_DAYS": {
+    "long_name": "Nombre de jours de neige et de jours de chute de neige durant les jours fériés",
+    "description": "Nombre de jours de neige et de jours de chute de neige durant les jours fériés.",
+    "title": "Jours de neige et de chute de neige durant les jours fériés",
+    "abstract": "Nombre de jours de neige et de jours de chute de neige durant les jours fériés."
```

```suggestion
    "long_name": "Nombre de jours de neige durant les jours de Noël",
    "description": "Nombre de jours de neige durant les jours de Noël.",
    "title": "Jours de neige durant les jours de Noël",
    "abstract": "Nombre de jours de neige durant les jours de Noël."
  },
  "HOLIDAY_SNOW_AND_SNOWFALL_DAYS": {
    "long_name": "Nombre de jours de neige et de jours de chute de neige durant les jours de Noël",
    "description": "Nombre de jours de neige et de jours de chute de neige durant les jours de Noël.",
    "title": "Jours de neige et de chute de neige durant les jours de Noël",
    "abstract": "Nombre de jours de neige et de jours de chute de neige durant les jours de Noël."
```
Pas certain ici. "durant les fêtes" ? "durant Noël" ?
« Jours fériés » includes any day without work, it's a bit too general.